docs/core/components/datatable.component.md

```diff
@@ -448,15 +448,16 @@ Learn more about styling your datatable: [Customizing the component's styles](#c
 
 ### Events
 
-| Name                | Type                                                                                                                                                            | Description                                             |
-|---------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------|
-| columnOrderChanged  | [`EventEmitter`](https://angular.io/api/core/EventEmitter)`<`[`DataColumn`](../../../lib/core/src/lib/datatable/data/data-column.model.ts)`<>[]>`               | Emitted when the column order is changed.               |
-| columnsWidthChanged | [`EventEmitter`](https://angular.io/api/core/EventEmitter)`<`[`DataColumn`](../../../lib/core/src/lib/datatable/data/data-column.model.ts)`<>[]>`               | Emitted when the column width is changed.               |
-| executeRowAction    | [`EventEmitter`](https://angular.io/api/core/EventEmitter)`<`[`DataRowActionEvent`](../../../lib/core/src/lib/datatable/components/data-row-action.event.ts)`>` | Emitted when the user executes a row action.            |
-| rowClick            | [`EventEmitter`](https://angular.io/api/core/EventEmitter)`<`[`DataRowEvent`](../../../lib/core/src/lib/datatable/data/data-row-event.model.ts)`>`              | Emitted when the user clicks a row.                     |
-| rowDblClick         | [`EventEmitter`](https://angular.io/api/core/EventEmitter)`<`[`DataRowEvent`](../../../lib/core/src/lib/datatable/data/data-row-event.model.ts)`>`              | Emitted when the user double-clicks a row.              |
-| showRowActionsMenu  | [`EventEmitter`](https://angular.io/api/core/EventEmitter)`<`[`DataCellEvent`](../../../lib/core/src/lib/datatable/components/data-cell.event.ts)`>`            | Emitted before the actions menu is displayed for a row. |
-| showRowContextMenu  | [`EventEmitter`](https://angular.io/api/core/EventEmitter)`<`[`DataCellEvent`](../../../lib/core/src/lib/datatable/components/data-cell.event.ts)`>`            | Emitted before the context menu is displayed for a row. |
+| Name                      | Type                                                                                                                                                            | Description                                             |
```

Comment:
Here similarly. Please compare with table above that there formatting looks better: 
<img width="1661" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/eba75e25-a292-4394-9814-3759c684dd2f">